docs/prow/manage-component-jobs-with-templates.md

```diff
@@ -14,122 +12,148 @@ Follow these steps:
 
 1. Edit the configuration file.
 
-Go to `templates/config.yaml` and add a new entry with your component details to the `render` list under the `templates` section.
+   Go to `templates/data/generic_component_data.yaml` and add a new entry with your component details to the `render` list under the `templates` section.
+   
+   See an example that defines the `compass-runtime-agent` component from the `kyma` repository, using the generic bootstrap:
 
-See an example that defines the `application-broker` component from the `kyma` repository, using the generic bootstrap:
+   ```yaml
+   templates:
+   - from: generic.tmpl
+     render:
+      - to: ../../prow/jobs/kyma/components/compass-runtime-agent/compass-runtime-agent-generic.yaml
+        jobConfigs:
+         - repoName: "github.com/kyma-project/kyma"
+           jobs:
+            - jobConfig:
+              path: components/compass-runtime-agent
+              args:
+              - "/home/prow/go/src/github.com/kyma-project/kyma/components/compass-runtime-agent"
+              run_if_changed: "^components/compass-runtime-agent/|^common/makefiles/"
+              release_since: "1.7"
+              optional: true
+           ...
+   ```
 
-```yaml
-  - from: templates/generic-component.yaml
-    render:
-      - to: ../prow/jobs/kyma/components/application-broker/application-broker-generic.yaml
-        values:
-          <<: *kyma_generic_component
-          path: components/application-broker
-          since: "1.7"
-          optional: true
-    ...
-```
+   Such an entry uses the `generic.tmpl` template to create the `compass-runtime-agent-generic.yaml` file under the `/prow/jobs/kyma/components/compass-runtime-agent/` subfolder, specifying that the presubmit and postsubmit jobs for this component should apply from the `1.7` release onwards.
+   Set the **optional** parameter to `true` for this job to be optional on pull requests (PRs), not to block others.
 
-Such an entry uses the `generic-component.yaml` template to create the `application-broker-generic.yaml` file under the `/prow/jobs/kyma/components/application-broker/` subfolder, specifying that the presubmit and postsubmit jobs for this component should apply from the `1.7` release onwards. Set the **optional** parameter to `true` for this job to be optional on pull requests (PRs), not to block others.
+   If needed, you can add global Config Sets (**globalSets**) to the `templates/config.yaml` file.
 
-> **NOTE:** Make sure that the `.yaml` file and the component folder name are the same as the name of the Kyma component. Also, all `.yaml` files in the whole `jobs` structure need to have unique names.
+   - For more information about creating template file, as well as local config sets (**localSets**), job configs (**jobConfig**) and (**globalSets**), read [Render Templates](https://github.com/kyma-project/test-infra/tree/main/development/tools/cmd/rendertemplates).
```

Comment:
```suggestion
   For more information about creating template files, as well as local config sets (**localSets**), job configs (**jobConfig**) and (**globalSets**), read [Render Templates](https://github.com/kyma-project/test-infra/tree/main/development/tools/cmd/rendertemplates).
```

There's no second bullet in this list, so we can remove the bullet. I hope I didn't mess up the indentation :D